{
  "FaultLabel": "Resolved power cut",
  "Type": "ResolvedFault",
  "Icon": "icon-success",
  "ShowViewOnMapCTA": false,
  "UnderReview": false,
  "AdditionalFaultInfo": "Good news! The electricity in your area has been restored. The power cut was caused by an unexpected incident on our underground cable that provides your supply.",
  "CTAs": [],
  "faultNumber": "40792456",
  "multipleFaultNumbers": null,
  "date": "2020-12-31T17:08:16",
  "region": "Thornton-Cleveleys",
  "faultType": "Resolved Fault",
  "faultStatus": "Resolved",
  "previousFaultStatus": "Resolved",
  "consumersOff": 0,
  "estimatedTimeOfRestoration": "2021-01-01T01:00:00",
  "estimatedTimeOfRestorationMajority": "2020-12-31T17:08:16",
  "information": "",
  "actualTimeOfRestoration": "2020-12-31T17:08:16",
  "outageCentrePoint": {
    "lat": 53.879626,
    "lng": -3.019029
  },
  "outageLocations": null,
  "AffectedPostcodes": " FY5 2ET, FY5 2EU, FY5 2EY, FY5 2EZ, FY5 2ST",
  "EtrMUnderReview": true,
  "WebTMSFaultType": "U/GM"
}